guidelines/sc/21/text-spacing.html

```diff
@@ -15,10 +15,8 @@ <h4>Text Spacing</h4>
  
  <p>Exception: Human languages and scripts that do not make use of one or more of these text style properties in written text can conform using only the properties that exist for that combination of language and script.</p>
 
- <p class="note">It is not required that content use the values specified. The requirement is to ensure no loss of content or functionality when a user overrides the authored styles with the values.</p>
+ <p class="note">Content is not required to use these text spacing values. The requirement is to ensure that <strong>when a user overrides</strong> the authored text spacing, content or functionality is not lost.</p>
 
-<p class="note">Some human languages or writing systems have different needs and may adjust other values (such as paragraph start indent) in order to improve legibility. This success criterion might not be sufficient for those needs.</p>
-
-<p class="note">It is beneficial for users if authors use any locally available guidance for improving readability in the local language or writing system. If the user chooses to go beyond the metrics specified, any resulting loss of content or functionality is the user's responsibility.</p>
+ <p class="note">Some language writing systems do not use all of the text spacing settings in this success criterion. Some use different types of spacing to improve readability and legibility, such as paragraph start indent.</p>
```

Comment:
"Some... do not use all..." seems a little unclear. I also am not fond of the "language writing systems" term, since I tend to be allergic to _more_ jargon 😉.

```suggestion
<p class="note">Some languages or <a>writing systems</a> do not use one or more of the text spacing settings in this success criterion. In addition, some will use different types of spacing to improve readability or legibility, such as paragraph start indent.</p>
```

Reply:
RE: "language writing systems". To try to simplify the discussion here, I put a question for you at https://docs.google.com/document/d/1BoPr_AXLoSTAvcm6OOGVm-2FmbIKC7rKTifhnxucQ6Q/edit#heading=h.5ofxatogf2uf

Reply:
humm. "...use one or more of the text spacing settings in this success criterion. In addition, some will..." seems like extra wording that's not necessary? Though I think it's not wrong.

Reply:
Please see edited PR to try to succinctly address key points.